buildout.cfg

```diff
@@ -9,6 +9,7 @@ parts =
 [versions]
 Products.ZCatalog =
 RestrictedPython = >= 5.1
+zope.interface = >= 7.2
```

Comment:
Without this line `pip` installs 7.2 but `buildout` wants to install 7.1.1 (as pinned in  https://zopefoundation.github.io/Zope/releases/master/versions.cfg). This fails on MacOS  for known reasons: buildout cannot handle the universal wheels.

It's a pitty that we have to have it here … maybe updating the master versions would be a better solution.

Reply:
There's no reason you couldn't just update the pin in Zope and then update the gh-pages branch. I am sure it won't break anything.

Reply:
Done in https://github.com/zopefoundation/Zope/pull/1241

Reply:
So I guess you can remove that pin now.

Reply:
I updated the Zope gh-pages branch right now so when the new pin is published everything should be OK

Reply:
Thank you. I removed the pin now.